File: starter-bots/NETCoreBot/NETCoreBot/Models/PlayerActionDto.cs
using System;
using NETCoreBot.Enums;

namespace NETCoreBot.Models
{
    public class PlayerActionDto
    {
        public Guid TargetNodeId { get; set; }
        public int NumberOfUnits { get; set; }
        public int TickActionCompleted { get; set; }
        public int TickActoinStart { get; set; }
        public int TickReceived { get; set; }
        public ActionType ActionType { get; set; }
    }
}

File: starter-bots/JavaBot/src/main/java/Models/Dtos/PlayerActionDto.java
package Models.Dtos;

import Enums.ActionTypes;

import java.util.UUID;

public class PlayerActionDto {
    public UUID TargetNodeId;
    public int NumberOfUnits;
    public int ExpectedCompletedTick;
    public int StartTick;
    public ActionTypes ActionType;

}
